arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -217,8 +217,8 @@ bool parse_ngram_vpack_config(const irs::string_ref& args, irs::analysis::ngram_
     return false;
   }
 
-  if (!slice.hasKey(PRESERVE_ORIGINAL_PARAM_NAME) ||
-      !slice.get(PRESERVE_ORIGINAL_PARAM_NAME).isBool()) {
+  if (!slice.hasKey(PRESERVE_ORIGINAL_PARAM_NAME.c_str()) ||
```

Comment:
change `PRESERVE_ORIGINAL_PARAM_NAME` to `string_view`?

Reply:
velocypack::StringRef than. As it mostly used with vpack